File: cs50_x/week6/me/agree.py
# Simple terms and conditions program.

def main():
    # Gets input from standard input.
    answers = ["y", "n", "yes", "no"]
    while True:
        answer = input("Do you agree?[y/n] ").lower()
        if answer in answers:
            break

    # Prints output to standard output.
    if answer in ["yes", "y"]:
        print("User has agreed.")
    if answer in ["no", "n"]:
        print("User has not agreed.")

main()

File: cs50_x/week6/me/agree.c
// Simple terms and conditions program.

#include <stdio.h>

int main(void)
{
    // Gets input from standard input.
    puts("Do you agree?[y/n]");
    char answer;
    do
        scanf("%c", &answer);
    while (answer != 'n' && answer != 'N' && answer != 'y' && answer != 'Y');

    // Prints output to standard output.
    if (answer == 'y' || answer == 'Y')
        puts("User has agreed.");
    else if (answer == 'n' || answer == 'N')
        puts("User has not agreed.");
}